.github/workflows/tethys-release.yml

```diff
@@ -76,13 +77,14 @@ jobs:
           docker push ${{ secrets.DOCKER_UPLOAD_URL }}:latest;
 
   conda-build:
-    name: Conda Build (${{ matrix.platform }})
+    name: Conda Build (${{ matrix.platform }}, ${{ matrix.django-version }}, ${{ matrix.python-version }})
     runs-on: ${{ matrix.platform }}
     strategy:
       fail-fast: false
       matrix:
         platform: [ubuntu-latest]
-        python-version: [3.10]
+        python-version: ["3.10", "3.11", "3.12"]
+        django-version: ["3.2", "4.2", "5.0"]
```

Comment:
I don't think that the matrix is necessary with the conda build. We probably just want to generate a package with python and django unpinned.

Reply:
done